src/setup.cfg

```diff
@@ -36,12 +36,20 @@ console_scripts =
 
 [flake8]
 max-line-length = 119
-exclude = migrations,static,build,dist,sdist
+exclude =
+  migrations
+  static
+  build
+  dist
+  sdist
 
 [isort]
 include_trailing_comma = true
 known_first_party = chaosinventory
 multi_line_output = 5
 line_length = 79
 honor_noqa = true
-skip = bootstrap,chaosinventory/settings.py
+skip =
+  bootstrap
```

Comment:
Where does "bootstrap" come from?

Reply:
*insert head scratching here* I removed it and it kept working ^^

Thanks for the heads-up!